control-mode.el

```diff
@@ -195,6 +200,13 @@ Control mode is a global minor mode."
          (setq control-mode-rebind-to-shift v)
          (control-mode-reload-bindings)))
 
+(defcustom global-control-mode-exceptions '()
+  "List of modes to exclude for `global-control-mode'."
+  :group 'control
+  :type '(repeat (function))
+  :set (lambda (x v)
+         (setq global-control-mode-exceptions v)))
```

Comment:
I could avoid this setter by just adding the defcustom where the defvar is now. But given how you do it for the other defcustom I guess you want it this way?